documentation/docs/steps/neoDeploy.md

```diff
@@ -54,3 +54,38 @@ steps:
       account: <myDeployAccount>
       host: hana.example.org
 ```
+
+## Example for invalidating the cache
+
+Setting parameter `invalidateCache` to `true`, will clean up the cache of the deployed application and reflects the new changes.
+Setting this parameter to true, needs additional configurations to go through. Firstly, create an oauth credential as shown below:
```

Comment:
I believe we can help the customer by guiding her through each single step of the way, i.e.:
1. SAP Cloud Platform
2. Left hand side panel choose Security/OAuth
3. Choose Clients tab in OAuth Settings
4. Click Register New Client

If the image 'oauthClientCreation.png' could also show the left hand panel with Security/OAuth settings would be great too, but not a must.